src/keyring/KeyringController.test.ts

```diff
@@ -144,6 +170,14 @@ describe('KeyringController', () => {
     );
   });
 
+  it('should not export account if address is not provided', async () => {
+    await expect(
+      async () => await keyringController.exportAccount(password, ''),
+    ).rejects.toThrow(
+      'No keyring found for the requested account. Error info: The address passed in is invalid/empty; There are keyrings, but none match the address;',
+    );
+  });
+
```

Comment:
maybe another test(s) `should not export account if password is not provided` and/or `should not export account if password provided is incorrect`>

Reply:
yep added